jar-infer/src/main/java/com/ibm/wala/examples/util/ExampleUtil.java

```diff
@@ -0,0 +1,33 @@
+package com.ibm.wala.examples.util;
+
+import com.ibm.wala.ipa.callgraph.AnalysisScope;
+import com.ibm.wala.util.config.FileOfClasses;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+
+public class ExampleUtil {
```

Comment:
Check if this file was originally under a permissive OSS license and if so, consider renaming it and re-licensing it under the same license as the rest of NullAway. Otherwise we need to do without it.

Reply:
I am actually not sure this file is useful at all, since most of the classes below are not going to be on the jars we analyze and we are not doing interproc...